install/kubernetes/cilium/templates/cilium-agent/daemonset.yaml

```diff
@@ -251,8 +251,12 @@ spec:
         resources:
           {{- toYaml . | trim | nindent 10 }}
         {{- end }}
-        {{- if or .Values.prometheus.enabled (or .Values.hubble.metrics.enabled .Values.hubble.metrics.dynamic.enabled) }}
         ports:
+        - name: health
+          containerPort: {{ .Values.healthPort }}
+          hostPort: {{ .Values.healthPort }}
+          protocol: TCP
+        {{- if or .Values.prometheus.enabled (or .Values.hubble.metrics.enabled .Values.hubble.metrics.dynamic.enabled) }}
```

Comment:
nit: I think this guard should be re-evaluated.

Since this PR introduce a named port unconditionally, guarding each port independently depending on config is enough.  I'd suggest to remove this condition and matching end (new line 288), then add a new guard around the `peer-service` port `{{- if .Values.hubble.enabled }}`.

Reply:
I think I've done what you requested.  Can you double check?